NAPS2.Lib/EtoForms/Ui/DownloadProgressForm.cs

```diff
@@ -51,7 +51,7 @@ protected override void BuildLayout()
     protected override void OnLoad(EventArgs e)
     {
         base.OnLoad(e);
-        Controller.Start();
+        Controller.StartDownloadsAsync();
```

Comment:
This is probably not the best way to do things and Visual Studio warns that we do not await it. If there is a way to have an async "OnLoad" let me know and I'll improve this.